### PR TITLE
added a page with data indexers, included PARSIQ data provider in it

### PR DESCRIPTION
**Pull request by Ivan-Ivanitskiy**
Added a page with data indexers, included PARSIQ data provider in it.
image

### CODE REVIEW

**Comment by symbolpunk** [comment]
Hi @Ivan-Ivanitskiy - thanks for the contribution. Would you mind reformatting this as a third-party doc following [the latest docs contribution guidelines](https://docs.arbitrum.io/for-devs/third-party-docs/contribute)?

If we find that our readers need general guidance on data indexers, we'll produce a core reference doc that includes PARSIQ. In the meantime, this could be a PARIQ-specific how-to, quickstart, or concept doc that helps readers understand how PARSIQ can help them as they build on Arbitrum. Eg:

```
📂 Third-party content
  📂 PARSIQ
    📄 (a how-to, quickstart, or concept doc here)
```

**Comment by Ivan-Ivanitskiy** [comment]
Hey @symbolpunk ! Thank you for the feedback. I've reverted the changes and added PARSIQ Quickstart to third-party-docs, as recommended. Please take a look and have a great day!
image

File: arbitrum-docs/for-devs/third-party-docs/PARSIQ/parsiq.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: "Get blockchain data with PARSIQ"
+description: "Learn how to use PARSIQ API and SDK to get Arbitrum data"
+author: Ivan-Ivanitskiy
+sme: Ivan-Ivanitskiy
+sidebar_label: "PARSIQ"
+---
+
+# PARSIQ Data API & SDK Quickstart
```
Comment (symbolpunk) [inline arbitrum-docs/for-devs/third-party-docs/PARSIQ/parsiq.md:9]:
```suggestion
```

File: arbitrum-docs/for-devs/third-party-docs/PARSIQ/parsiq.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
+title: "Get blockchain data with PARSIQ"
```
Comment (symbolpunk) [inline arbitrum-docs/for-devs/third-party-docs/PARSIQ/parsiq.md:2]:
```suggestion
title: "Quickstart: PARSIQ (blockchain data API & SDK)"
```

File: arbitrum-docs/for-devs/third-party-docs/PARSIQ/parsiq.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+sidebar_label: "PARSIQ"
+---
+
+# PARSIQ Data API & SDK Quickstart
+
+**[<ins>PARSIQ</ins>](https://www.parsiq.net/)** is a reliable, fully customizable blockchain data indexer, helping developers to seamlessly access, process and utilize Web3 data - both raw and custom.
+
+PARSIQ API allows querying blockchain data such as transactions, token transfers, events, internal function calls, blocks, etc.
+
+PARSIQ SDK allows accumulationg, processing, and storing this data according to the developer-specified logic, as well as building your own API endpoints.
```
Comment (symbolpunk) [inline arbitrum-docs/for-devs/third-party-docs/PARSIQ/parsiq.md:15]:
```suggestion
PARSIQ SDK allows accumulating, processing, and storing this data according to the developer-specified logic, as well as building your own API endpoints.
```

File: arbitrum-docs/for-devs/third-party-docs/PARSIQ/parsiq.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+Some more complicated cases where custom data needs to be stored, accumulated, and calculated, cannot be covered by an API. In that cases, use PARSIQ SDK or go for a Custom Data Lake. They allow you to set up data bases and data processing logic to solve your specific use case. 
+
+:::info SDK Documentation
+
+Please see [<ins>PARSIQ SDK documentation</ins>](https://docs.parsiq.net/reference/your-own-web3-api) for more details.
+
+:::
+
+---
```
Comment (symbolpunk) [inline arbitrum-docs/for-devs/third-party-docs/PARSIQ/parsiq.md:43]:
```suggestion
```

**Review by symbolpunk — CHANGES_REQUESTED** [review]
Some minor nits @Ivan-Ivanitskiy, otherwise LGTM! Thanks for the contribution!

**Review by symbolpunk — APPROVED** [review]

**Pull request merged**